corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -80,12 +80,10 @@ def locations(self):
     @property
     def sql_location(self):
```

Comment:
[Not blocking] It may be nice to give `OTARestoreUser` the same API as `CouchUser`, that is: `user.get_sql_location(domain)` and `user.get_sql_locations(domain)`.